src/modules/helpfulFunctions.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-restricted-globals */
+// src/modules/helfulFunctions
+
+// function to help validate input
+const validateScore = (name, score) => {
+  if (name === null || name === '') {
+    return false;
+  }
+  if (isNaN(score)) {
+    return false;
+  }
```

Comment:
- [ ] Please try to avoid disabling ESlint errors as they are there to improve your coding standards and ensure you are not writing code that goes against the coding standard or code that might lead to bugs in this case you simply need to prefix `isNaN` with the class `Number`. so something like `Number.isNaN(score)` will avoid you having to disable the linter warning. please implement this small change.